pkg/preparer/orchestrate_test.go

```diff
@@ -89,7 +95,7 @@ func (f *fakeHooks) RunHookType(hookType hooks.HookType, pod hooks.Pod, manifest
 		f.ranAfterLaunch = true
 		return f.afterLaunchErr
 	case hooks.AFTER_AUTH_FAIL:
-		f.ranAfterAuthFail = true
+		f.ranAfterAuthFail <- struct{}{}
```

Comment:
The test would deadlock if they had not anticipated the auth failure, right? Seems like something you'd see in the ensuing panic.